src/ui/Distance/DistanceResources.ts

```diff
@@ -61,91 +61,113 @@ export class DistanceResources extends Component {
    */
   public static options: IDistanceOptions = {
     /**
-     * Specifies the field that will contain the distance value.
+     * Specifies the name of the field in which to store the distance value.
      *
      * Specifying a value for this option is required for the `DistanceResources` component to work.
      */
     distanceField: ComponentOptions.buildFieldOption({
       required: true
     }),
+
     /**
-     * Specifies the field that contains the latitude value.
+     * Specifies the name of the field that contains the latitude value.
```

Comment:
Maybe we could add an explanation about what happens if you do not have a @latitude field in your index ? (a query error/exception).

Reply:
Sure! 👍